### ISSUE TITLE
attempting to deploy to rinkeby using mnemonic phrase but get "HardhatError: HH110: Invalid JSON-RPC response received: invalid project id"

### ISSUE THREAD

**Issue by SwineCoder101** [nomiclabs/hardhat#2063]
npx hardhat run scripts/deploy.js --network rinkeby
-------------------------------------------------------------------------

require("@nomiclabs/hardhat-waffle");
require("@nomiclabs/hardhat-ethers");
require("@nomiclabs/hardhat-etherscan");
require('hardhat-deploy');
// This is a sample Hardhat task. To learn how to create your own go to
// https://hardhat.org/guides/create-task.html
task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
  const accounts = await hre.ethers.getSigners();

  for (const account of accounts) {
    console.log(account.address);
  }
});

// You need to export an object to set up your config
// Go to https://hardhat.org/config/ to learn more
require('dotenv').config();
const RINKEBY_RPC_URL = process.env.RINKEBY_RPC_URL;
const MNEMONIC = process.env.MNEMONIC;
const PRIVATE_KEY = process.env.PRIVATE_KEY;

/**
 * @type import('hardhat/config').HardhatUserConfig
 */
module.exports = {
  defaultNetwork: "hardhat",
  networks: {
    hardhat: {

    },
    rinkeby: {
      url: RINKEBY_RPC_URL,
      // accounts: [PRIVATE_KEY, PRIVATE_KEY],
      accounts: {
        mnemonic: MNEMONIC
      },
      saveDeployments: true,
    }
  },
  etherscan: {
    // Your API key for Etherscan
    // Obtain one at https://etherscan.io/
    apiKey: process.env.ETHERSCAN_API_KEY
  },
  namedAccounts: {
    deployer: {
      default: 0,
      "rinkeby": '0x6c96Ab2e710360073a847a30f993261be830dC3C',
    }
  },
  paths: {
    artifacts: './react-dap/src/artifacts',
  },
  solidity: {
    compilers: [
      {
        version: "0.8.0"
      },
      {
        version: "0.7.0"
      },
      {
        version: "0.6.6"
      },
      {
        version: "0.4.24"
      }
    ],
  }
}

**Comment by alcuadrado** [comment]
This is a problem with your node provider, most probably infura.

**Comment by Btadford** [comment]
What was the solution to this? I am getting this exact error. How was infura the issue?

**Comment by devlancer412** [comment]
You need to add infura id to your rpc url
such as 
`https://rinkeby.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161`

**Comment by landafta** [comment]
Hi there,
Ater 30 minutes of furstration googling, trial and error, I've found the answer. The issue is with your testnet RPC_URL provider (Alchemy or Infura). You must first make sure that everything is running smooth on your local network (hardhat or whatever). Then, if you put in an RPC_URL provided by Alchemy and you get this error "HardhatError: HH110: Invalid JSON-RPC response received:" switch to Infura and it will work (that's what I did). Same if you get this issue with Infura switch to Achemy and see the result.
